packages/manager/apps/ips/src/pages/listing/index.tsx

```diff
@@ -69,10 +77,9 @@ export default function Listing() {
               isSelected={location.pathname === tab.to}
               className="flex items-center justify-center"
               title={tab.title}
+              onClick={() => onTabClick(tab.to)}
```

Comment:
```suggestion
              onClick={() => navigate(tab.to)}
```

onTabClick does not prevent re-render here so no need to create this function

Reply:
ok